src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs

```diff
@@ -6,22 +6,26 @@
 namespace Microsoft.AspNet.Mvc.ModelBinding
 {
     /// <summary>
-    /// Interface for model binding.
+    /// Defines an interface for model binders.
```

Comment:
Standard for .NET API docs of interfaces usually takes the form "Represents a ..", so in this case e.g. "Represents an object that performs model binding operations.", or some such.